### PR TITLE
I wanted to write a fun lil ruby program

### PR DESCRIPTION
**Pull request by briantgale**

### CODE REVIEW

File: newfile.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This is a fun ruby file 
+
+while true do
+  puts "I like pie"
+end
```
Comment (codyolsen) [inline newfile.rb:5]: yeah pretty sure this a 🍰  loop...

**Review by codyolsen — CHANGES_REQUESTED** [review]
Please add correct filename and structure for ruby architecture.

File: newfile.rb (hunk 1 above)
Comment (briantgale) [reply newfile.rb:5]: Nope.

Comment (codyolsen) [reply newfile.rb:5]: MKay

**Review by codyolsen — APPROVED** [review]

**Pull request merged**